spec.md

```diff
@@ -1740,6 +1780,9 @@ message ControllerServiceCapability {
       // SINGLE_NODE_SINGLE_WRITER and/or SINGLE_NODE_MULTI_WRITER are
       // supported, in order to permit older COs to continue working.
       SINGLE_NODE_MULTI_WRITER = 13 [(alpha_enum_value) = true];
+
+      // Indicates the SP supports volume group snapshot
+      VOLUME_GROUP_SNAPSHOT = 14 [(alpha_enum_value) = true];
```

Comment:
does this make more sense in the capabilities of the GroupControllerService? i.e. if there is no GroupControllerService, would it make sense for the ControllerService to advertise this cap? (IMO, no)

Reply:
This is the place where we have the intersection. I added this for the optional field `group_snapshot_id` in message Snapshot which is in the ControllerService. Can the `group_snapshot_id` field in the ControllerService be dependent on a capability in GroupControllerService?

Reply:
Removed VOLUME_GROUP_SNAPSHOT from the ControllerService.